### ISSUE TITLE
Released the binarys individually

### ISSUE THREAD

**Issue by BolverBlitz** [gekware/minecraft-server-hibernation#141]
### Is there an existing feature request for this?

- [X] I have searched the existing issues before opening this feature request.

### Describe the feature you would like to see.

Currently you provide a .zip for every release, please release all 3 seperate so they an be downloaded from github directly in a script.

### Describe the solution you'd like.

Not upload a .zip

**Comment by gekigek99** [comment]
done

**Comment by BolverBlitz** [comment]
Please remove
```json
    "Whitelist": [
      "{127.0.0.1}",
      "{gekigek99}"
    ]
``` 
from the released config file, this brings up the error "You don´t have permissions to start the server" when freshly installed.

**Comment by gekigek99** [comment]
fixed in 6c34c8ee0fd42af84ae85ec8e37e3737523fd3c9
updated release
